db/schema.rb

```diff
@@ -11,18 +11,18 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20141229224941) do
+ActiveRecord::Schema.define(version: 20150306182232) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
   create_table "active_admin_comments", force: :cascade do |t|
-    t.string   "namespace",     limit: 255
```

Comment:
Resetting the db can be problematic.  Personally I've found that the more you use it, the more likely that there will be migration or other db issues later.  I always prefer `drop`, `create`, `migrate`, `seed`.  I haven't heard that anywhere but I've noticed that.